### DISCUSSION TITLE
MQT tele log message 17:55:14.419 MQT: stat/heatpad/RESULT = {"Command":"Unknown"}

### DISCUSSION THREAD

**Discussion by pmendiuk** [arendst/Tasmota#18069] [Q&A]
I am using the PID driver with GenericPOWR2 and I would like to find the cause of this repetitive console log entry.  

18:05:04.460 MQT: tele/heatpad/SENSOR = {"Time":"2023-02-27T18:05:04","ENERGY":{"TotalStartTime":"2021-05-16T20:51:21","Total":69.643,"Yesterday":0.000,"Today":0.044,"Period":0,"Power":0,"ApparentPower":0,"ReactivePower":0,"Factor":0.00,"Voltage":0,"Current":0.000},"DS18B20":{"Id":"011921482719","Temperature":71.9},"PID":{"PidPv":22.25,"PidSp":22.20,"PidEnable":1,"PidPb":1.00,"PidTi":1800.00,"PidTd":15.00,"PidInitialInt":0.50,"PidDSmooth":3.00,"PidAuto":1,"PidManualPower":0.00,"PidMaxInterval":300,"PidUpdateSecs":0,"PidPower":0.25},"TempUnit":"F"}
18:05:04.470 MQT: stat/heatpad/RESULT = {"POWER":"ON"}
18:05:04.475 MQT: stat/heatpad/POWER = ON
18:05:04.485 MQT: stat/heatpad/RESULT = {"Command":"Unknown"}
18:05:08.419 MQT: stat/heatpad/RESULT = {"POWER":"OFF"}
18:05:08.423 MQT: stat/heatpad/POWER = OFF

Where is that log entry generated?

**Comment by sfromis** [comment]
To debug what happens, try again after enabling `Weblog 3` for details. That should show you what command from where cause the message.

**Reply by pmendiuk** [reply]
Thanks,
I now see what caused it:

18:40:54.432 CMD: Group 0, Index 1, Command "SHUTTERPOSITION", Data "20"
18:40:54.441 MQT: stat/heatpad/RESULT = {"Command":"Unknown"}
18:40:54.571 CFG: Saved to flash at F7, Count 1383117, Bytes 4096

**Reply by sfromis** [reply]
Unrelated, `Count 1383117` looks like you may be wearing your flash chip down by very frequent saves.

**Comment by pmendiuk** [comment]
It seems every log error operation is getting saved.  How do I reduce/eliminate all those writes to flash?

**Reply by sfromis** [reply]
Not using shutters, I'm not sure how much is covered by `SetOption0` which controls saving power state when changed.
You can also use `SaveData` to set a longer interval after changes than the default of 1 second, or disable saving.

Of course, If Tasmota is restarted for some reason (including power outages), not saving state may mean being out of sync with the physical shutter after restart. Again, not sure about details for handling such.

**Comment by pmendiuk** [comment]
It looks like SetOption0 worked.  I wanted to recompile without shutters but somehow that PlatformIO project now has a bunch of errors in ESP8266Audio after a PlatformIO update.
